projects/packages/jetpack-mu-wpcom/package.json

```diff
@@ -48,9 +49,11 @@
 	"dependencies": {
 		"@automattic/calypso-color-schemes": "3.1.3",
 		"@automattic/color-studio": "4.0.0",
+		"@automattic/components": "2.2.0",
```

Comment:
This also brings in `i18n-calypso`, which may give us trouble with i18n (see the comment below). Are there components in here that we need without equivalents in `@wordpress/components` or `@automattic/jetpack-components`?